docs/specs/xref.yml

```
@@ -2162,3 +2162,69 @@ outputs:
   .errors.log: |
     {"message_severity":"warning","code":"missing-attribute","message":"Missing required attribute: 'name'.","file":"docs/TOC.md"}
     {"message_severity":"warning","code":"xref-not-found","message":"Cross reference not found: 'b'.","file":"docs/TOC.md"}
+---
+# files to be redirected must be excluded from xref map
+noSingleFile: true
+repos:
+  https://docs.com/redirected-uid-excluded-from-xrefa#master:
+    - files:
+        docfx.yml:
+        _themes/ContentTemplate/schemas/TestData.schema.json: |
+          {
+            "renderType": "component",
+            "properties": {
+              "uid": { "contentType": "uid" },
+              "fullName": { "type": "string" },
+              "name": { "type": "string" },
+              "title": { "type": "string" }
+            },
+            "xrefProperties": ["title","name","fullName"]
+          }
+        a.yml: |
+          ### YamlMime:TestData
+          uid: a
+          title: Title a.yml
+          name: name a
+          fullName: fullName a
+        b.yml: |
+          ### YamlMime:TestData
+          uid: b
+          title: Title b
+          name: name b
+          fullName: fullName b
+        redirections.yml: |
+          redirections:
+            a.yml: /absolute/path2
+outputs: 
+  .xrefmap.json: | 
+    {
+      "references":[
+        {
+          "uid": "b",
+          "title": "Title b",
+          "name": "name b",
+          "fullName": "fullName b"
+        }
+      ]
+    }
+  b.json: | 
```

Ignore the content of this file. The purpose of the test is to check xrefmap, can simply assert `a` does not exit like so:
```suggestion
  .xrefmap.json: |
     { "references" : [{ "uid": "a" }] }
  b.json:
```